open-craft/opencraft
# [BB-2326] OCIM Bugs

Issue: [BB-2326](https://tasks.opencraft.com/browse/BB-2326)

Testing Instructions:
* First Bug:
Try to login with opened Redux/React Dev tools(to have better clarity on states) and check for the console state being refreshed or not

* Second Bug:
Added a Go To Console button in congratulation Page, just create a new instance and click on Go To Console button to check if it's working or not.
Screenshot: ![go_to_console](https://user-images.githubusercontent.com/8607745/80759396-de38ce80-8b54-11ea-94d2-19e69c1b4997.png)

* Third Bug:
Added a Django atomic operation for Theme and static updates.

Reviewers:
